.travis.yml

```diff
@@ -1,10 +1,13 @@
 language: node_js
+
 notifications:
   email: false
+
 before_script:
   - npm prune
-after_success:
-  - npm run semantic-release
-branches:
-  except:
-    - /^v\d+\.\d+\.\d+$/
+
+deploy:
+  provider: script
+  skip_cleanup: true
+  script:
+    - npm run semantic-release
```

Comment:
Do we need to scope this to master?

```yaml
  on:
    branch: master
```

Reply:
The Semantic Release recipe for Travis CI don't include this.
https://github.com/semantic-release/semantic-release/blob/caribou/docs/recipes/travis.md
I'm not confident but was hoping to check the build logs to confirm in Travis CI.

Reply:
The deployment step is already scoped to the default branch by default
> Skipping a deployment with the script provider because this branch is not permitted
https://travis-ci.org/seek-oss/sku/builds/350665289#L3740

I've added on: branch master just to be sure.